frontend/src/app/shared/components/show-changes/show-changes.component.ts

```diff
@@ -0,0 +1,251 @@
+  @HostBinding('class.op-show-changes') className = true;
+
+  public opened = false;
+
+  public dropDownDescription = '';
+
+  public nonWorkingDays:IDay[] = [];
+
+  public selectedDate = '';
+
+  public selectedTimezoneFormattedTime = '';
+
+  public filterSelected = false;
+
+  public timeZoneSelected = false;
+
+  public daysNumber = 0;
+
+  public tooltipPosition = SpotDropAlignmentOption.BottomRight;
+
+  public text = {
+    toggle_title: this.I18n.t('js.show_changes.toggle_title'),
+    header_description: this.I18n.t('js.show_changes.header_description'),
+    clear: this.I18n.t('js.show_changes.clear'),
+    apply: this.I18n.t('js.show_changes.apply'),
+    show_changes_since: this.I18n.t('js.show_changes.show_changes_since'),
+    time: this.I18n.t('js.show_changes.time'),
+    help_description: this.I18n.t('js.show_changes.help_description'),
+    timeZone: this.configuration.isTimezoneSet() ? moment().tz(this.configuration.timezone()).zoneAbbr() : 'local',
+    time_description: () => this.I18n.t('js.show_changes.time_description', { time: this.selectedTimezoneFormattedTime, days: this.daysNumber }),
+  };
+
+  public showChangesAvailableValues = [
+    {
+      value: '0',
+      title: this.I18n.t('js.show_changes.drop_down.none'),
+    },
+    {
+      value: '1',
+      title: this.I18n.t('js.show_changes.drop_down.yesterday'),
+    },
+    {
+      value: '2',
+      title: this.I18n.t('js.show_changes.drop_down.last_working_day'),
+    },
+    {
+      value: '3',
+      title: this.I18n.t('js.show_changes.drop_down.last_week'),
+    },
+    {
+      value: '4',
+      title: this.I18n.t('js.show_changes.drop_down.last_month'),
+    },
+    {
+      value: '5',
+      title: this.I18n.t('js.show_changes.drop_down.a_specific_date'),
+    },
+    {
+      value: '6',
+      title: this.I18n.t('js.show_changes.drop_down.between_two_specific_dates'),
+    },
+  ];
+
+  public query$ = this.wpTableFilters.querySpace.query.values$();
+
+  constructor(
+    readonly I18n:I18nService,
+    readonly wpTableFilters:WorkPackageViewFiltersService,
+    readonly halResourceService:HalResourceService,
+    private weekdaysService:WeekdayService,
+    private daysService:DayResourceService,
+    readonly timezoneService:TimezoneService,
+    private configuration:ConfigurationService,
+  ) {
+    super();
+  }
+
+  async ngAfterViewInit():Promise<void> {
+    await this.requireNonWorkingDaysOfTwoYears();
+  }
+
+  public toggleOpen():void {
+    this.opened = !this.opened;
+  }
+
+  public clearSelection():void {
+  }
+
+  public onSubmit(e:Event):void {
+    e.preventDefault();
+
+    this.close();
+  }
+
+  public close():void {
+    this.opened = false;
+  }
+
+  public yesterdayDate():string {
+    const today = new Date();
+    this.daysNumber = -1;
+
+    today.setDate(today.getDate() - 1);
+    this.selectedDate = moment(today).format('YYYY-MM-DD');
+    return moment(today).format('ddd, YYYY-MM-DD');
```

Comment:
Please use the `TimezoneService` `formattedDate` `formattedDatetime` to get the configured date/time format to show instead of ISO8601